src/Symfony/Component/HttpClient/DataCollector/HttpClientDataCollector.php

```diff
@@ -59,7 +59,8 @@ public function lateCollect()
 
             $this->data['clients'][$name]['traces'] = array_merge($this->data['clients'][$name]['traces'], $traces);
             $this->data['request_count'] += \count($traces);
-            $this->data['error_count'] += $this->data['clients'][$name]['error_count'] += $errorCount;
+            $this->data['error_count'] += $errorCount;
```

Comment:
Global error_count is wrong because on real lateCollect, we readd all errors to the counter.